### ISSUE TITLE
csportfolio

### ISSUE THREAD

**Issue by Schlegelo** [JTLiggett/csPortfolio#2]
I like how you have organized your projects, it's very clean and the difficulty rating is a nice touch. Also nice job giving a breakdown of your code.

**Comment by VotoA** [comment]
I agree, but the organization of the page could be a little more in order and there's also a couple typos in your reflections. Overall though, Great job, especially on your dice.
